cmd/vulcan-prowler/main.go

```diff
@@ -325,16 +363,31 @@ func fillCisLevelVuln(v *report.Vulnerability, r *prowlerReport, alias string, s
 			if err != nil {
 				return nil, err
 			}
+			cinfo, ok := controls[control]
+			if !ok {
+				return nil, fmt.Errorf("no information for control %s", control)
+			}
 			row := map[string]string{
-				"Control":     control,
-				"Description": description,
-				"Region":      e.Region,
-				"Message":     e.Message,
+				"Control":      control,
+				"Description":  description,
+				"CIS Severity": cinfo.SeverityLiteral,
+				"Region":       e.Region,
+				"Message":      e.Message,
+				"Remediation":  fmt.Sprintf("<a href=\"%s\">Reference</a>", cinfo.Remediation),
 			}
-			fcTable.Rows = append(fcTable.Rows, row)
+			c := controlRow{row, cinfo.Severity}
+			rows = append(rows, c)
+			fallthrough
+		default:
+			total++
 		}
 	}
-
+	sort.Slice(rows, func(i, j int) bool {
+		return rows[i].score > rows[j].score
```

Comment:
Just a silly thing, but in this case the order won't be the same in different executions. I'd put more cases for the sort, for example sorting by the control number if the score is the same.

Reply:
That's a very good idea indeed thanks!!